lib/features/groups/pages/pdf/yearly_report.dart

```diff
@@ -40,7 +41,13 @@ class _YearlyReportState extends State<YearlyReport> {
   void initState() {
     super.initState();
     _group = widget.group;
-
+    _startDate = currentDate.subtract(const Duration(days: 365));
+    _endDate = currentDate;
+    dtchange = DateTimeRange(
+        start: DateTime(
+          _startDate.year,
```

Comment:
unwanted variable 

you can use start and end dt 

and for use as
initialValue: DateTimeRange(start:_startDate, end:_endDate)